.github/workflows/docker-publish-full.yml

```diff
@@ -28,6 +28,7 @@ env:
 
 jobs:
   build_and_test:
+    if: ${{ ! startsWith(github.head_ref, '^dependabot/*') }} && ${{ ! github.event.pull_request.head.repo.fork }}
```

Comment:
I think we do not need to skip the job in this case. Running the Docker integration tests, without uploading to GHCR, should work fine, even on PRs from non-members.

Reply:
I think https://github.com/daq-tools/lorrystream/pull/19 may be the correct solution.